src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -145,6 +143,18 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         group.getPermissions().clear();
                         MessageUtils.sendSuccessMessage("Cleared all permissions from the group: " + group.getName(), channel);
                         return;
+                    } else if (args[2].equalsIgnoreCase("move")) {
+                        if(args.length >= 4) {
+                            int pos = GeneralUtils.getInt(args[3], -1);
```

Comment:
Set this to 1 by default, we may as well just serve the first page rather than dealing with them having to do another command.

Reply:
Thats for move so if i set the default to 1 the perms will be moved to the top if the number is invalid. I don't think we want that

Reply:
Oh then you need to fix the string below, 
`"Invalid Page: " + args[3]`

Reply:
Whops. Guess thats my fault for working with pages so much